apiaudio/api_resources/script.py

```diff
@@ -1,3 +1,4 @@
+from ast import arg
```

Comment:
VS code 🤡

Reply:
oh its a pain - need more C++ and xcode :P